boards/atxmega-a1u-xpro/Makefile.dep

```diff
@@ -1,4 +1,5 @@
 USEMODULE += boards_common_atxmega
+USEMODULE += periph_ebi
```

Comment:
```suggestion
FEATURES_REQUIRED += periph_ebi
```

But even better would be adding `FEAUTERS_OPTIONAL += periph_ebi` at cpu-level.

The reason is that always both the feature and the corresponding module implementing it should be used, as inconsistency between them can result in broken dependency resolution. By just always pulling in the feature, the corresponding modules will be automatically included.

Reply:
It was changed to `atxmega_ebi` module. It is resolved automatically at `cpu/atxmega`.